### ISSUE TITLE
Use f-strings in places that pylint recommends

### ISSUE THREAD

**Issue by mtlynch** [tiny-pilot/tinypilot#995]
New versions of pylint flag places where we can use [Python f-strings](https://realpython.com/python-f-strings/) instead of the legacy string formatting options.

We currently have pylint's check for this disabled because of our legacy code that uses the old formatting style, but we should replace those instances with modern f-strings and delete [the suppression in .pylintrc](https://github.com/tiny-pilot/tinypilot/blob/d89da5e8513fbf15b3af8b231b0522ff1350fc25/.pylintrc#L26).

**Comment by ezinneanne** [comment]
Hello, can I be assigned to this?

**Comment by mtlynch** [comment]
Sure, that would be great!

This depends on #993, which is not yet merged into master, but you can fork from the `pylint-2.14.2` branch and make these fixes.

**Comment by mtlynch** [comment]
@ezinneanne - #993 is now merged, so you don't have to do anything special with branch forking. You can open a regular PR against the `master` branch.

**Comment by jmunoz94** [comment]
Hey, @mtlynch: I see that this issue is assigned, but it has been a while and it seems there is no progress on it (ex: ezinneanne has not even forked the repository). Can I get this assigned to me/work on this?

**Comment by mtlynch** [comment]
Sure, @jmunoz94 - assigned.
